databases/week1/hw-week1.sql

```diff
@@ -0,0 +1,47 @@
+
+select * from task ;
+-- task 1
+select count(*) as tasks_count from task;
+
+-- task 2
+select title from task
+where due_date is null;
+
+ -- task 3
+select title from task
+where status_id = 3;
+
+-- task 4
+select title from task
+where status_id = 1 and 2;
+
+
+-- task 5
+select * from task
+order by created desc;
+
+-- task 6
+select title from task
+order by created desc;
```

Comment:
here we get all the tasks, but we need to show the single most recently created task.
you can take a look at LIMIT clause here (https://www.mysqltutorial.org/mysql-limit.aspx)

Reply:
i applied limit 3